hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/tracing/Span.java

```diff
@@ -17,28 +17,54 @@
  */
 package org.apache.hadoop.tracing;
 
+import io.opentelemetry.context.Scope;
+
 import java.io.Closeable;
 
 public class Span implements Closeable {
-
+  private io.opentelemetry.api.trace.Span span = null;
```

Comment:
1. add some javadoc to line 24 to warn this wraps the opentelemetry span.
2. skip the =null assignment, as it will save the jvm from some needless init. assuming a lot of spans get created, this may matter
3. give the field a different name. like 'openspan'